pandas/tests/io/excel/test_readers.py

```diff
@@ -148,6 +149,30 @@ def parser(self, *args, **kwargs):
             expected = expected_defaults[read_ext[1:]]
         assert result == expected
 
+    def test_engine_kwargs(self, read_ext, engine):
+        # GH#52214
+        expected_defaults = {
+            "xlsx": {"foo": "abcd"},
+            "xlsm": {"foo": 123},
+            "xlsb": {"foo": "True"},
+            "xls": {"foo": True},
+            "ods": {},
```

Comment:
Can you add the args for ods - this currently isn't being tested.

Reply:
Done!